lib/browser.js

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import bedrock from 'bedrock';
+import 'bedrock-express';
+import onHeaders from 'on-headers';
+
+bedrock.events.on('bedrock-express.init', app => {
+  app.use((req, res, next) => {
+    onHeaders(res, function() {
+      const setCookie = res.getHeader('set-cookie');
+      const userAgent = req.headers['user-agent'];
+      if(!setCookie) {
+        return;
+      }
```

Comment:
We don't need to fetch `userAgent` if `set-cookie` isn't there, so skip that. Also, we want to use express's `req.get` function when retrieving headers, as it normalizes case and all that.

```suggestion
      if(!setCookie) {
        return;
      }
      const userAgent = req.get('user-agent');
```